packs/feats/crystal-luminescence.json

```diff
@@ -25,7 +25,40 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Impossible Lands"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "TokenLight",
+                "predicate": [
+                    "crystal-luminescence"
+                ],
+                "value": {
+                    "animation": {
+                        "type": "torch"
+                    },
+                    "bright": "20+@item.flags.pf2e.rulesSelections.crystalLuminescence",
+                    "color": "#1e0ab8",
```

Comment:
We should give people the choice of color when taking the feat. See the rule elements on Effect: Spirit Power (Passion)

If we need a new color added to the system standard colors we should also add it to that effect and Lantern Surki.

Reply:
Colour selection added by https://github.com/foundryvtt/pf2e/pull/15116/commits/4e936841d4ade577aa2d7612a36fb9bc73caaa4b